src/helpers/validator.js

```diff
@@ -0,0 +1,41 @@
+import * as Yup from "yup";
+
+export const schema = Yup.object().shape({
+  nume: Yup.string().required("required"),
+  adresa_localitate: Yup.string().required("required"),
+  adresa_judet: Yup.string().required("required"),
+  adresa_strada: Yup.string().required("required"),
```

Comment:
This can be also empty (I'm thinking about small villages with numbers only, without streets).

Reply:
removed street address validation